ci/twiggy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+mkdir master
+cd master
+git clone https://github.com/yewstack/yew
+cd ../
+cargo build --all --release --target wasm32-unknown-unknown
+cd master/yew
+cargo build --all --release --target wasm32-unknown-unknown
+cd ../../
+mkdir twiggy_results
+for filename in target/wasm32-unknown-unknown/release/*.wasm; do
+  localname=$(basename $filename)
+  echo "Comparing old $localname against new $localname"
+  touch twiggy_results/$localname.csv
```

Comment:
I've never used twiggy so I don't know this, but does it really require that the file already exist before writing to it?
You probably already tested this but I just want to make sure.